sklearn/metrics/_scorer.py

```diff
@@ -469,11 +469,20 @@ def _check_multimetric_scoring(estimator, scoring):
         The estimator for which the scoring will be applied.
 
     scoring : list, tuple or dict
-        A single string (see :ref:`scoring_parameter`) or a callable
-        (see :ref:`scoring`) to evaluate the predictions on the test set.
+        Strategy to evaluate the performance of the cross-validated model on
+        the test set.
 
-        For evaluating multiple metrics, either give a list of (unique) strings
-        or a dict with names as keys and callables as values.
+        If `scoring` represents a single score, one can use:
+
+        - a single string (see :ref:`scoring_parameter`);
+        - a callable (see :ref:`scoring`) that returns a single value.
+
+        If `scoring` reprents multiple scores, one can use:
+
+        - a list or tuple of unique string;
```

Comment:
`_check_multimetric_scoring` only accepts a list tuple or dict. It expects the caller to handle the callable and the string case (The single metric cases).

This was done because in the callable case, one does not know the names of the metrics before the function is called. For grid search, the scorer is first called when computing the test scores.

That being said, we should still include the list of options for a list, tuple, or dict.